docs/development/build-instructions-windows.md

```diff
@@ -39,21 +39,32 @@ there is no Visual Studio project generated.
 
 ```powershell
 $ cd electron
-$ python script\bootstrap.py -v
+$ npm run bootstrap
+```
+
+You can also bootstrap for debug builds:
+
+```powershell
+$ cd electron
+$ npm run bootstrap:dev
```

Comment:
Same suggestion as above for explaining `developer` and non-developer bootstrapping, and for not referring to debug bootstraps